src/schema/v2/sorts/show_sorts.ts

```diff
@@ -5,6 +5,7 @@ const SHOW_SORTS = {
   END_AT_DESC: { value: "-end_at" },
   FEATURED_ASC: { value: "featured" },
   FEATURED_DESC: { value: "-featured" },
+  FEATURED: { value: "-featured,-end_at" },
```

Comment:
Should this be `FEATURED_DESC_END_AT_DESC`?

Reply:
Renamed!